examples/terraform/vsphere/variables.tf

```diff
@@ -157,13 +157,25 @@ variable "control_plane_memory" {
   type        = number
 }
 
+variable "control_plane_num_cpus" {
+  default     = 2
+  description = "number of cpus of each control plane node"
+  type        = number
+}
+
 variable "worker_memory" {
   default     = 2048
   description = "memory size of each worker node in MB"
   type        = number
 
 }
 
+variable "worker_num_cpus" {
+  default     = 2
+  description = "number of cpus of each workers node"
+  type        = number
+}
+
```

Comment:
Just a nit; can we please implement this for `vsphere-flatcar` as well? I know it's benign but I'd love to keep it somewhat consistent.

Reply:
That's a good point. Fixed.